plugins/available/cmd-returned-notify.plugin.bash

```diff
@@ -0,0 +1,16 @@
+# shellcheck shell=bash
```

Comment:
Wouldn't this rather belong in a separate PR?

Reply:
Maybe, this plugin was why i needed to update preexec in the first place so I thought I would sneak it in. :wink: 
I suppose it could be PR'ed afterwards as well. How ever I also needed to test that the updated preexec library could run more functions per event. 
I am good either way

Reply:
I am okay with including it in this PR